common/web/gesture-recognizer/src/engine/touchEventEngine.ts

```diff
@@ -3,7 +3,17 @@ import { InputEventEngine } from "./inputEventEngine.js";
 import { InputSample } from "./headless/inputSample.js";
 import { Nonoptional } from "./nonoptional.js";
 import { ZoneBoundaryChecker } from "./configuration/zoneBoundaryChecker.js";
+import { GestureSource } from "./headless/gestureSource.js";
 
+function touchListToArray(list: TouchList) {
+  const arr: Touch[] = [];
+
+  for(let i=0; i < list.length; i++) {
+    arr.push(list.item(i));
+  }
+
+  return arr;
+}
```

Comment:
I take it the spread operator doesn't work? `return [...list];`?

Reply:
It does not.  You can't even iterate over a `TouchList` with `for...of`, which is the entire reason I made the helper.